esbuild.config.js

```diff
@@ -42,11 +42,9 @@ const base = {
   inject: [`esbuild.react.shim.js`],
 };
 
-sources.map((source) => {
+Object.entries(sources).map(([name, source]) => {
   const opts = Object.assign({}, base);
-  opts.entryPoints = [`${inDir}${source}`];
-  opts.outfile = `${outDir}${path
-    .basename(source)
```

Comment:
this line would take `foundation/pages/mozfest/index.js` and turn it into `index.js`, which broke the load instruction for `mozfest.compiled.js` as static asset.